website/static/files/build_orca_reconcile.sql

```diff
@@ -6,150 +6,147 @@ SET search_path TO orca, public;
+  );
+  COMMENT ON TABLE reconcile_job IS 'Manages internal reconciliation job information.';
+  COMMENT ON COLUMN reconcile_job.id IS 'Job ID unique to each internal reconciliation job.';
+  COMMENT ON COLUMN reconcile_job.orca_archive_location IS 'ORCA S3 Glacier bucket the reconciliation targets.';
+  COMMENT ON COLUMN reconcile_job.status_id IS 'Current status of the job.';
+  COMMENT ON COLUMN reconcile_job.inventory_creation_time IS 'Inventory report creation time from the s3 manifest.';
+  COMMENT ON COLUMN reconcile_job.start_time IS 'Date and time the internal reconcile job started.';
+  COMMENT ON COLUMN reconcile_job.last_update IS 'Date and time the job status was last updated.';
+  COMMENT ON COLUMN reconcile_job.end_time IS 'Time the job completed and wrote the report information.';
+  COMMENT ON COLUMN reconcile_job.error_message IS 'Critical error the job ran into that prevented it from finishing.';
 
-    -- Create reconcile_s3_objects table
-    CREATE TABLE IF NOT EXISTS reconcile_s3_objects
-    (
-      job_id                  int8 NOT NULL
-    , orca_archive_location   text NOT NULL
-    , key_path                text NOT NULL
-    , etag                    text NOT NULL
-    , last_update             timestamp with time zone NOT NULL
-    , size_in_bytes           int8 NOT NULL
-    , storage_class           text NOT NULL
-    , CONSTRAINT PK_reconcile_s3_objects_1 PRIMARY KEY(job_id,orca_archive_location,key_path)
-    , CONSTRAINT reconcile_jobs_s3_objects_fk FOREIGN KEY(job_id) REFERENCES reconcile_jobs(id)
-    );
 
-    COMMENT ON TABLE reconcile_s3_objects
-      IS 'Temporary table that holds the listing from the ORCA S3 bucket to use for comparisons against the ORCA catalog.';
-    COMMENT ON COLUMN reconcile_s3_objects.job_id
-      IS 'Job the S3 listing is a part of for the comparison. Foreign key to the reconcile jobs table.';
-    COMMENT ON COLUMN reconcile_s3_objects.orca_archive_location
-      IS 'ORCA S3 Glacier bucket name where the file is stored.';
-    COMMENT ON COLUMN reconcile_s3_objects.key_path
-      IS 'Full path and file name of the object in the S3 bucket from the S3 list command.';
-    COMMENT ON COLUMN reconcile_s3_objects.etag
-      IS 'AWS etag value from the S3 list command.';
-    COMMENT ON COLUMN reconcile_s3_objects.last_update
-      IS 'AWS Last Update from the S3 list command.';
-    COMMENT ON COLUMN reconcile_s3_objects.size_in_bytes
-      IS 'AWS size of the file in bytes from the S3 list command.';
-    COMMENT ON COLUMN reconcile_s3_objects.storage_class
-      IS 'AWS storage class the object is in from the S3 list command.';
+  CREATE TABLE IF NOT EXISTS reconcile_s3_object  ( 
+    job_id               	int8 NOT NULL
+  , orca_archive_location	text NOT NULL
+  , key_path             	text NOT NULL
+  , etag                 	text NOT NULL
+  , last_update          	timestamp with time zone NOT NULL
+  , size_in_bytes        	int8 NOT NULL
```

Comment:
should size_in_bytes be renamed to s3_size just like in reconcile_catalog_mismatch_report table?

Reply:
in that case, it has to be updated in the architecture drawing and other places.

Reply:
Since it's under the s3 table, I think adding `s3` would be redundant, but I absolutely agree that we have some naming inconsistencies here. Switching s3_size to s3_size_in_bytes.

Reply:
Renamed `s3_size` to `s3_size_in_bytes`